ls1intum/Artemis
# Fix errors when the exercise of the exercise details header is undefined

### Checklist

### Motivation and Context

Fixes the same issue as #3314. For some reason that code is duplicated (which I unfortunately didn't notice right away), so we have to fix this file as well.

### Steps for Testing

1. Find or create an exercise with categories
2. Navigate to the instructor dashboard for the course and from there to the exercise (instructor) dashboard of the exercise with categories
3. Check the console for `TypeError: this.exercise is undefined`

Example: https://artemistest.ase.in.tum.de/course-management/306/instructor-dashboard/7456
